index.js

```diff
@@ -67,8 +69,11 @@ function runSequence(gulp) {
 	}
 
 	function finish(e) {
+		if (finished) return;
```

Comment:
Finished appears to never be set.

Reply:
Also, please match the style of this codebase, no spaces between language keywords (`if`, `catch`, etc) and the parentheses.